yt/yt/library/auth_server/config.h

```diff
@@ -232,6 +250,12 @@ class TOAuthServiceConfig
     std::optional<TString> UserInfoSubjectField;
     std::optional<TString> UserInfoErrorField;
 
+    //! If specified, configures a global regex replace transformation on the
+    //! contents of the login field. It will replace all non-overlapping matching
+    //! strings with the replacement string. You can use \1-\9 for captured match
+    //! groups in the replacement string, and \0 for the whole match.
+    TStringReplacementConfigPtr LoginTransformation;
```

Comment:
I would suggest making this field `std::vector<>` from the very beginning to support various replacements applied in order.